plugin/src/test/java/org/opensearch/ml/action/controller/GetControllerTransportActionTests.java

```diff
@@ -5,9 +5,7 @@
 
 package org.opensearch.ml.action.controller;
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
Generally we should avoid star imports.

Reply:
ah good catch, IDE automatically made this changes. Will update my IDE settings.